### DISCUSSION TITLE
Background-image doesn't appear

### DISCUSSION THREAD

**Discussion by jonatsuji** [neuefische/questions#398] [Web] [answered]
Hi everyone,

I try to implement a bg-image to one of my pages with styled components. The bg-image also includes a gradient and the problem is now, that I can see the gradient from the image but not the image itself. The data format is .jpg, is that a problem?
I checked the import and also the picture itself and all seems correct to me, maybe someone finds a problem here?
See the screenshots or my fitting [PR](https://github.com/jonatsuji/capstone-project/pull/2
![Screenshot 2022-12-15 at 11 13 13](https://user-images.githubusercontent.com/115465230/207834361-c5fe299f-cf7f-49f5-8d66-8300df3cf87f.png)
![Screenshot 2022-12-15 at 11 14 29](https://user-images.githubusercontent.com/115465230/207834373-2ce90799-8dc5-4789-aa4e-ec0eab714127.png)
).

Many thanks 🙏

**Comment by MerleMak** [comment]
Hi Jonathan,
I think if you move your assets folder into your public folder, this could fix the problem :) Let me know!

**Reply by jonatsuji** [reply]
@MerleMak unfortunately not, I moved the folder and changed the path correctly but the image still doesn't appear

**Comment by MerleMak** [comment]
Okay, also the { } around your imported image are unnecessary :)

**Reply by jonatsuji** [reply]
That didn't fix it aswell but we found the solution. Apparently you are not allowed to import the image and use a ${} to define the src. 
The path must be inserted directly into the styled component.

thanks for your help

**Comment by MerleMak** [comment, marked as answer]
Great that it works! 
Another option that works would be :

```js
import backgroundImage from '../../assets/images/sandstone-bright-crack.jpg';
...
const StyledPageContainer = styled.div`
  background-image: linear-gradient(
      to bottom,
      rgba(245, 246, 252, 0.52),
      rgba(117, 19, 93, 0.73)
    ),
    url('${backgroundImage.src}');

**Comment by MerleMak** [comment]
Also you can always use the Image component from Next, a big argument for that is the optimisation of the image size, so it loads more quickly :)

**Reply by Elgin-Schultz** [reply]
Dear Merle, I tried to do that yesterday but I read that the Next image component does not work for background pictures. You have to do a workaround with Z-index instead ... See link: https://uharston.medium.com/next-js-image-optimization-on-background-images-65de18ea03f5
